dashboard/app/assets/stylesheets/application.scss

```diff
@@ -3694,4 +3694,8 @@ html[dir=rtl] {
   }
 }
 
+.validation-children div > p {
```

Comment:
these styles are only used on the frontend, so they should be in a frontend SCSS file